packaging/suse/patches/0_change_versions.gem.patch

```diff
@@ -1,16 +1,16 @@
 diff --git a/Gemfile b/Gemfile
-index cd17fc91be2a..cd5d5580f36d 100644
+index 3e66267d110e..6bf7db01fec7 100644
 --- a/Gemfile
 +++ b/Gemfile
 @@ -1,6 +1,6 @@
  source "https://rubygems.org"
  
 -gem "rails", "~> 4.2.8"
 +gem "rails", "~> 4.2.2"
- gem "jquery-rails"
```

Comment:
this red refers to the 0_change_versions.gem.patch, not to the PR. Sorry. This is good